src/prompt-input/index.tsx

```diff
@@ -19,8 +19,8 @@ const PromptInput = React.forwardRef(
       spellcheck,
       readOnly,
       actionButtonIconName,
-      minRows,
-      maxRows,
+      minRows = 1,
```

Comment:
it'd be good to remove the default values from the InternalPromptInput too, so that they're only set in one place